### ISSUE TITLE
Change model_data to model

### ISSUE THREAD

**Issue by KateFriedman-NOAA** [NOAA-EMC/GSI#761]
By NCO's request, `model_data` should be just `model` in COM paths.

The following instances are observed within the GSI:
```
sorc/gsi_enkf.fd/regression/global_enkf.sh:     $nln $datens/$memchar/model_data/atmos/history/${prefix_ens}.atmf00${fhr}.nc sfg_${global_adate}_fhr0${fhr}_${memchar}
sorc/gsi_enkf.fd/regression/global_enkf.sh:            $nln $datens/$memchar/model_data/atmos/history/${prefix_ens}sfcf00${fhr}.nc sfgsfc_${global_adate}_fhr0${fhr}_${memchar}
sorc/gsi_enkf.fd/regression/global_enkf.sh:    $nln $datens/ensstat/model_data/atmos/history/${prefix_ens}.atmf00${fhr}.ensmean.nc sfg_${global_adate}_fhr0${fhr}_ensmean
sorc/gsi_enkf.fd/regression/global_4denvar.sh:dathis=$COMROOTgfs/$dumpges.$PDYg/${cycg}/model_data/atmos/history
sorc/gsi_enkf.fd/regression/global_4denvar.sh:$nln $datens/ensstat/model_data/atmos/history/${prefix_ens}.sfcf006.ensmean.nc         ./sfcf06_anlgrid
sorc/gsi_enkf.fd/regression/global_4denvar.sh:  $nln $datens/$member/model_data/atmos/history/$sigens ${ENS_PATH}sigf${fh}_ens_${member}
```

Companion issue in workflow: https://github.com/NOAA-EMC/global-workflow/issues/2686

**Comment by RussTreadon-NOAA** [comment]
Thank you @KateFriedman-NOAA for opening this GSI issue.  Changes are needed to GSI scripts and test case data directories.

Attention @ShunLiu-NOAA , @hu5970 , and @CoryMartin-NOAA

**Comment by RussTreadon-NOAA** [comment]
Work for this issue will be done in [RussTreadon-NOAA:feature/model](https://github.com/RussTreadon-NOAA/GSI/tree/feature/model).

**Comment by RussTreadon-NOAA** [comment]
`regression/regression_var.sh` specifies the following paths for directory which contains data to run global and regional GSI regression (ctests):
- Gaea:  `export casesdir="/gpfs/f5/ufs-ard/world-shared/GSI_data/CASES/regtest"`
- WCOSS2:  `export casesdir="/lfs/h2/emc/da/noscrub/russ.treadon/CASES/regtest"`
- Orion & Hercules: `export casesdir="/work/noaa/da/rtreadon/CASES/regtest"`
- Hera: `export casesdir="/scratch1/NCEPDEV/da/Russ.Treadon/CASES/regtest"`
- Jet: `export casesdir="/lfs1/NESDIS/nesdis-rdo2/David.Huber/save/CASES/regtest"`
- Discover: `export casesdir="/discover/nobackup/projects/gmao/obsdev/wrmccart/NCEP_regression/CASES"`

The following has been done on WCOSS2 (Cactus and Dogwood), Orion (Hercules), and Hera:
1. `cd $casesdir/gfs/prod/enkfgdas.20240222/18`
2. in each directory `ensstat`, `mem001`, `mem002`, up to `mem010`
    - `mv model_data model`
    - `ln -fs model model_data` 
3. `cd cd $casesdir/gfs/prod/gdas.20240222/18`
    - `mv model_data model`
    - `ln -fs model model_data`

Once PR #762 is merged into `develop`, the link `model_data` will be removed from the WCOSS2, Orion/Hercules, and Hera `casesdir`.

We need to also `mv model_data model` on Gaea, Jet, and Discover.  @DavidHuber-NOAA and @DavidBurrows-NCO , can you make this change on Jet and Gaea, respectively, when the time comes?

@ShunLiu-NOAA, @hu5970 , @CoryMartin-NOAA , and @wx20jjung :  Do we need to maintain a GSI ctest capability on Discover?  Who runs the GSI on Discover?

**Comment by wx20jjung** [comment]
Russ,

Thanks for the note.  I no longer use Discover.
Jim

**Comment by DavidBurrows-NCO** [comment]
@RussTreadon-NOAA No problem. I can take care of Gaea when the time comes.

**Comment by DavidHuber-NOAA** [comment]
@RussTreadon-NOAA `/lfs1` is no longer a valid mount on Jet, so the `cases` directory will need to be recreated elsewhere.  I will rsync the data from Hera into `/lfs5/NESDIS/nesdis-rdo2/David.Huber/save/CASES/regtest` and notify when complete.

**Comment by DavidHuber-NOAA** [comment]
@RussTreadon-NOAA The data have been synced from Hera to Jet and the links with them.  I have made changes to `regression_var.sh` and will open a PR to [feature/model](https://github.com/RussTreadon-NOAA/GSI/tree/feature/model) shortly.

**Comment by DavidBurrows-NCO** [comment]
Morning @RussTreadon-NOAA.  I see this PR was closed. Gaea is down today, so I will get those paths updated when it's back and let you know.

**Comment by RussTreadon-NOAA** [comment]
Thank you @DavidBurrows-NCO .  Yes, we can get path changes for Gaea and Jet global regression test data into future GSI PRs.

**Comment by DavidBurrows-NCO** [comment]
@RussTreadon-NOAA I updated the Gaea directories for GSI regression tests and tested the global runs:
```
Test project /gpfs/f5/ufs-ard/scratch/David.Burrows/gw_submods/gsi_my_fork/build
    Start 1: global_4denvar
    Start 6: global_enkf
1/2 Test #6: global_enkf ......................   Passed  664.40 sec
2/2 Test #1: global_4denvar ...................   Passed  2042.73 sec

100% tests passed, 0 tests failed out of 2

Total Test time (real) = 2042.79 sec
```

**Comment by RussTreadon-NOAA** [comment]
Thank you @DavidBurrows-NCO for updating the global paths on Gaea.  I appreciate your running global ctests to confirm that the tests work as they should.
